ui/webui/test/helpers/installer.py

```diff
@@ -40,6 +40,10 @@ class InstallerSteps(UserDict):
     _steps_jump[REVIEW] = PROGRESS
     _steps_jump[PROGRESS] = []
 
+    _parent_steps = {}
```

Comment:
I would suggest to not hardcode the prev pages, as they can easily be extracted from the steps_jumps. We use this already in the 'reach' method and can be introduced as helper.

prev_pages = [k for k, v in self.steps._steps_jump.items() if page in v]

Reply:
This is not related to order of pages, but to pages/steps which are substeps and therefore are not clickable before their parents.